src/Extensions/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -107,13 +107,13 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 				// get proper application name from host
 				serviceName = host.Services.GetService<IHostEnvironment>().ApplicationName;
 
-				ServiceInitializationEventSource.Instance.LogHostBuildSucceeded(Process.GetCurrentProcess().Id, serviceNameForLogging);
+				InitializationLogger.InitilizationSucceed(serviceNameForLogging);
```

Comment:
Typo. I would even change the name of the function completely as it sounds weird the way it is. It sounds more like a bool value than an action.
```suggestion
				InitializationLogger.InitializationSucceed(serviceNameForLogging);
```